src/components/Notifications/DownloadEventNotificationBody.tsx

```diff
@@ -0,0 +1,22 @@
+import { MantineColor, Progress, Text } from '@mantine/core';
+
+interface Props {
+  message: string;
+  downloadProgress: number;
+  color?: MantineColor;
+  animated?: boolean;
+}
+
+export function DownloadEventNotificationBody({
+  message,
+  downloadProgress,
+  color,
+  animated = true
+}: Props) {
+  return (
+    <>
+      <Text>{message}</Text>
```

Comment:
Using the `TruncatedText` component for the message body/title would make it so that we can hover over the text and see the full message :)  Please add for att messages and titles, since the notifications currently only show part of the titles and body

However, you might have to update the zIndex for the tooltips to make sure they are located over the notifications